docs/tutorials/submitting-speedrun.md

```diff
@@ -1,53 +1,56 @@
-
-## Framework
-
-We use the [FineWeb-Edu dataset](https://huggingface.co/datasets/HuggingFaceFW/fineweb-edu) by default. You can experiment with any model architecture, optimizers, learning rate schedules, or hyperparameters.
-[TODO (Nikil): The tokenized, shuffled dataset is also available on HF [link] to make speedrun more accessible]. We track the following key metrics:
-
-- **C4-EN BPB (Bits per Byte)**: Bits-per-byte on the `c4-en` (English portion of the C4) dataset; lower values are better.
-- **FLOPs Used**: Floating point operations used in training the model.
+Here is an [example submission](https://github.com/marin-community/marin/blob/main/experiments/speedrun/llama_75m_fineweb_edu/llama_75m_fineweb_edu.py),
+and here is the [current leaderboard](https://marin.community/speedrun).
 
+## Prerequisites
 
-While tracking performance on `c4-en` helps us understand model performance/capability, we opt to track a Pareto frontier to highlight trade-offs between model performance and scale. The Pareto frontier represents the set of models that are not outperformed in both metrics by any other submission. In other words, a model is Pareto-optimal if no other model achieves better BPB and uses fewer FLOPs.
+Before you get started, you will need the following:
 
-This approach encourages creative and efficient experimentation. For example:
+- Basic [installation](installation.md)
+- Set up your [local GPU](local-gpu.md)
+- You need to make sure that your Ray GPU cluster is started.
 
-- A large model with very low BPB but needing a lot of compute to train may be Pareto-optimal.
-- A smaller or more efficient model with worse BPB but dramatically lower compute may also make the frontier.
+## Framework
 
-The Pareto frontier should provide a clearer picture of what's achievable with different model scales and approaches, both in terms of model performance/capability, and the amount of compute it requires.
+You are given the [FineWeb-Edu dataset](https://huggingface.co/datasets/HuggingFaceFW/fineweb-edu),
+but you can define any training scheme you'd like.
+Each speedrun submission will generate the following key metrics:
 
+- **C4-EN BPB (Bits per Byte)** (quality): Bits-per-byte on the validation portion of the `c4-en` (English portion of the C4) dataset; lower values are better.
+- **FLOPs Used** (compute): Floating point operations used to train the model; lower values are better.
```

Comment:
should we say whether these are device or model flops

Reply:
I agree - @nikil-ravi and I chatted about this - I think it should be device FLOPs so you get rewarded for having good MFU

Reply:
Maybe we can patch this once we decide for sure